xc/xenops_server_xen.ml

```diff
@@ -2261,6 +2269,19 @@ module VGPU = struct
          Device.Dm.restore_vgpu task ~xs frontend_domid vgpu vcpus profile
       ) vm
 
+  let active_path vm vgpu = Printf.sprintf "/vm/%s/devices/vgpu/%s" vm (snd vgpu.Vgpu.id)
+
+  let set_active task vm vgpu active =
+    try
+      set_active_device (active_path vm vgpu) active
+    with e ->
+      debug "set_active %s.%s <- %b failed: %s" (fst vgpu.Vgpu.id) (snd vgpu.Vgpu.id) active (Printexc.to_string e)
+
+  let get_active vm vgpu =
```

Comment:
A better name would be "is_active" because this returns a boolean.

Reply:
I didn't rename this name which just wants to keep this behavior with the existing name in (VBD or VIF).
Thanks.